scene/animation/tween.cpp

```diff
@@ -318,6 +318,11 @@ bool Tween::should_pause() {
 }
 
 Variant Tween::interpolate_variant(Variant p_initial_val, Variant p_delta_val, float p_time, float p_duration, TransitionType p_trans, EaseType p_ease) {
+	if (p_duration == 0) {
+		// Special case to avoid dividing by 0 in equations.
+		return p_initial_val + p_delta_val;
```

Comment:
Doesn't work sadly:
```
 scene/animation/tween.cpp:323:24: error: use of overloaded operator '+' is ambiguous (with operand types 'Variant' and 'Variant')
                return p_initial_val + p_delta_val;
                       ~~~~~~~~~~~~~ ^ ~~~~~~~~~~~
```

Reply:
Ah right, this should've gone into easing equations .-.

Reply:
So I guess this is why I originally wanted to PR it after merging #50165 🤔